components/style/gecko_bindings/check_bindings.py

```diff
@@ -0,0 +1,32 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import re
+
+LICENSE = """/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Wouldn't that add a `\n` at the start of the generated file? Or the `\` removes the `\n` from the resulting string?